pipeline/git/git.go

```diff
@@ -30,5 +34,7 @@ func (Pipe) Run(ctx *context.Context) (err error) {
 		PreviousTag: previous,
 		Diff:        log,
 	}
+	// removes usual `v` prefix
+	ctx.Version = strings.TrimPrefix(tag, "v")
```

Comment:
If it's a requirement for Fpm we don't have a choice. 
But if a user only deploys to Github or Homebrew,  I think we should still allow other version formats. 
Maybe we can move the check to the fpm pipe?